src/redux/books/bookslice.js

```diff
@@ -1,26 +1,106 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/Mw6KlxQrMBAmuPW6nuKy/books';
+
+export const postBook = createAsyncThunk(
+  'books/postBook',
+  async (newBook) => {
+    try {
+      const response = await axios.post(url, newBook);
+      return response.data;
+    } catch (error) {
+      throw Error(error.response.data);
+    }
+  },
+);
+
+export const getBooks = createAsyncThunk(
+  'books/getBooks',
+  async () => {
+    try {
+      const response = await axios(url);
+      return response.data;
+    } catch (error) {
+      throw Error(error.response.data);
+    }
+  },
+);
```

Comment:
- [ ] Kindly dispatch a getBooks action that will populate your store with books coming from the API, you can put this in a useEffect hooks so that your App retrieves books from the API when the Application loads. Currently when I am refreshing the page I am loosing a the books created. Solve this and your App will be ready to go...